lib/mayaUsd/resources/ae/usdschemabase/ae_template.py

```diff
@@ -83,6 +83,25 @@ def onReplace(self, *args):
         # Nothing needed here since we don't create any UI.
         pass
 
+class UfeConnectionChangedObserver(ufe.Observer):
```

Comment:
Instead of creating a new one why can't you just add your "AttributeConnectionChanged" notif to the existing one and use that?

Reply:
That was actually the initial code I wrote, but the existing one seemed to be customized for transform, this is why I split into a second class that is more customized for shaders.